liquigraph-core/src/test/java/org/liquigraph/core/io/ChangelogGraphReaderTest.java

```diff
@@ -92,7 +85,34 @@ public void reads_changeset_with_multiple_queries() throws SQLException {
         }
     }
 
+    @Test
+    public void reads_changeset_with_multiple_queries_in_order() throws SQLException {
```

Comment:
Thanks for adding a test! I think you want here to test two executions in a row rather than one.
Otherwise, this test duplicates the previous one, unless I'm missing something.